test/modules/op/repeat.py

```diff
@@ -30,6 +31,27 @@ def get_example_inputs(self):
         return (torch.randn(1, 4),)
 
 
+@tag.use_onert
+class SimpleRepeatDynamicShape(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x):
+        z = x.repeat((128, 1, 1, 1))
+        return z
+
+    def get_example_inputs(self):
+        torch.manual_seed(1)
+        return (torch.randn(1, 4),)
+
+    def get_dynamic_shapes(self):
+        dim = Dim("dim", min=1, max=128)
+        dynamic_shapes = {
+            "x": {1: dim},
```

Comment:
it's just out of curiousity.
Why do you set key to `1`..?

Reply:
Because `dim0` is 128, which is not dynamic dimension..?